jetty9/src/main/docker/Dockerfile

```diff
@@ -35,13 +35,17 @@ ENV TMPDIR /tmp/jetty
 ADD jetty-base $JETTY_BASE
 ADD 15-debug-env-jetty.bash 50-jetty.bash /setup-env.d/
 ADD quickstart.sh /scripts/jetty/quickstart.sh
+ADD jetty_start.sh /scripts/jetty/jetty_start.sh
+ADD gen_start_command.sh /scripts/jetty/gen_start_command.sh
 WORKDIR $JETTY_BASE
 RUN mkdir -p webapps $TMPDIR \
  && java -jar $JETTY_HOME/start.jar --add-to-start=setuid \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR \
  && chmod +x /setup-env.d/15-debug-env-jetty.bash \
  && chmod +x /setup-env.d/50-jetty.bash \
- && chmod +x /scripts/jetty/quickstart.sh
+ && chmod +x /scripts/jetty/quickstart.sh \
+ && chmod +x /scripts/jetty/jetty_start.sh \
+ && chmod +x /scripts/jetty/gen_start_command.sh
 
 EXPOSE 8080
-CMD ["java","-Djetty.base=${jetty.base}","-jar","${jetty.home}/start.jar"]
+CMD ["/scripts/jetty/jetty_start.sh"]
```

Comment:
Some setup scripts (docker-entrypoint) in Jetty and Openjdk are expecting to see `java` in the command line to add specific arguments.  

https://github.com/GoogleCloudPlatform/jetty-runtime/blob/master/jetty9/src/main/docker/50-jetty.bash (At the end of the file )

https://github.com/GoogleCloudPlatform/openjdk-runtime/blob/master/openjdk8/src/main/docker/docker-entrypoint.bash (this one may not be affected)